Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN go test -mod=vendor -cover ./...
 RUN go build -mod=vendor -o /go/bin/app
 
 
-FROM gcr.io/distroless/static:966f4bd97f611354c4ad829f1ed298df9386c2ec
+FROM --platform=$BUILDPLATFORM gcr.io/distroless/static
```

Comment:
Thank you for fixing it!

question: Does it work if you keep the label? I prefer to pin the version rather than relying on the `latest`.

Reply:
I am not sure what the proper version would be for this since the repo they are using is not very ux friendly.